config/routes.rb

```diff
@@ -1,16 +1,18 @@
 Rails.application.routes.draw do
+  get 'bookmarks/show'
+  get 'bookmarks/new'
+  get 'bookmarks/edit'
 
-  resources :topics, only: [:new, :show, :create, :destroy]
-
+  resources :topics, only: [:new, :show, :create, :destroy, :index] do
+    resources :bookmarks, only: [:show, :new, :edit, :update, :create, :destroy]
```

Comment:
In this particular case, I think an argument could be made for `except`. You basically want everything but `index`, so this could be written more simply with `resources :bookmarks, except: [:index]`. I usually prefer `only`, but this seems to make sense here. Go with either option though, just thought I'd mention it! :)